plugin.yaml

```diff
@@ -1,5 +1,5 @@
 name: "update-config"
-version: "0.4.0"
+version: "0.4.1"
```

Comment:
Hi,

bumping version is also needed in this file https://github.com/zhanggbj/helm-update-config/blob/master/install.sh#L5, would you please update it and then I'll merge. Thanks!

Reply:
@subbarao-meduri ^^^^

Reply:
@zhanggbj Thanks for catching the omission, it is now fixed.